src/components/biggive-main-menu/biggive-main-menu.stories.tsx

```diff
@@ -0,0 +1,128 @@
+export default {
+  title: 'Components/Header and Footer',
+};
+
+const Template = () => `
+    <biggive-main-menu>
+        <div slot="social-icons">
+            <biggive-social-icon service="Facebook" url="https://www.facebook.com" background-colour="tertiary" icon-colour="black"></biggive-social-icon>
+            <biggive-social-icon service="Twitter" url="https://www.twitter.com" background-colour="tertiary" icon-colour="black"></biggive-social-icon>
+            <biggive-social-icon service="LinkedIn" url="https://www.linkedin.com" background-colour="tertiary" icon-colour="black"></biggive-social-icon>
+            <biggive-social-icon service="YouTube" url="https://www.youtube.com" background-colour="tertiary" icon-colour="black"></biggive-social-icon>
+            <biggive-social-icon service="Instagram" url="https://www.instagram.com" background-colour="tertiary" icon-colour="black"></biggive-social-icon>
+        </div>
+        <div slot="mobile-social-icons">
+            <biggive-social-icon service="Facebook" url="https://www.facebook.com" background-colour="tertiary" icon-colour="black"></biggive-social-icon>
+            <biggive-social-icon service="Twitter" url="https://www.twitter.com" background-colour="tertiary" icon-colour="black"></biggive-social-icon>
+            <biggive-social-icon service="LinkedIn" url="https://www.linkedin.com" background-colour="tertiary" icon-colour="black"></biggive-social-icon>
+            <biggive-social-icon service="YouTube" url="https://www.youtube.com" background-colour="tertiary" icon-colour="black"></biggive-social-icon>
+            <biggive-social-icon service="Instagram" url="https://www.instagram.com" background-colour="tertiary" icon-colour="black"></biggive-social-icon>
+        </div>
+        <ul class="links" slot="nav-primary">
```

Comment:
I don't know how much we'll mirror the story in real code, but could we give the lists or perhaps wrap in a surrounding `nav` with a suitable `role` for better accessibility? Might be good for the story to do this so we can check we're not depending upon the list being the first descendent.

https://www.a11yproject.com/posts/an-indepth-guide-to-aria-roles/

Reply:
@NoelLH They're already wrapped in a `nav` element when they're slotted in.

The nav is here: https://github.com/thebiggive/components/pull/99/files#diff-e74eba74290f4b51832f1dc4befbaab54e343ceb57edf8b4beacdfd0207dc1eeR92

And the `ul` gets slotted here: https://github.com/thebiggive/components/pull/99/files#diff-e74eba74290f4b51832f1dc4befbaab54e343ceb57edf8b4beacdfd0207dc1eeR111

Reply:
@NoelLH Let's me know if you still think that's insufficient.

Reply:
Ah got it! Nope that looks good @alihejazi